packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -252,20 +252,38 @@ class Viewport implements IViewport {
     vec3.scaleAndAdd(center, center, jVector, (size[1] / 2.0) * spacing[1]);
     vec3.scaleAndAdd(center, center, kVector, (size[2] / 2.0) * spacing[2]);
 
+    const imageToWorld: mat4 = [
+      direction[0],
+      direction[1],
+      direction[2],
+      0, //
+      direction[3],
+      direction[4],
+      direction[5],
+      0, //
+      direction[6],
+      direction[7],
+      direction[8],
+      0, //
+      0,
+      0,
+      0,
+      1, //
+    ];
+    const worldToImage: mat4 = mat4.transpose(
+      [1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 1, 2, 3, 4, 5, 6],
```

Comment:
I think you are creating an output array here, but can you just use api from `gl-matrix` ? or just put everything as 0

Reply:
you are right, we can simplify greatly the code